hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/HapiSpecRegistry.java

```diff
@@ -687,6 +693,11 @@ public FileID getFileId(String name) {
         return get(name, FileID.class);
     }
 
+    public long getNodeId(String name) {
```

Comment:
same here. I think you will be saving the `AccountID` sent in `NodeCreateTransactionBody` to a nodeId. So, `public long getNodeId(AccountID name)` ?

Reply:
Name is a convenience for hapi tests. That is how all the other hapi entities are saved.